test/tests/serial/sedfile.bats

```diff
@@ -36,38 +36,6 @@ function setup() {
   assert_output 'foo baz'
 }
 
-@test 'checking sedfile substitute failure (on first container start)' {
-  # delete marker
-  _run_in_container rm '/CONTAINER_START'
-  assert_success
-
-  # try to change 'baz' to 'something' and fail
-  _run_in_container sedfile -i 's|baz|something|' "${TEST_FILE}"
-  assert_failure
-  assert_output --partial "No difference after call to 'sed' in 'sedfile' (sed -i s|baz|something| /tmp/sedfile-test.txt)"
-
-  # file unchanged?
-  _run_in_container cat "${TEST_FILE}"
-  assert_success
-  assert_output 'foo bar'
-
-  # recreate marker
-  _run_in_container touch '/CONTAINER_START'
-  assert_success
-}
```

Comment:
Are you sure this test case should be removed?

It's not related to DMS restarts. You'll notice the start and end of the test case ensures `CONTAINER_START` is not present during the test, so `SKIP_ERROR` condition would not have applied here regardless. It ensures that `sedfile` should fail when expected to.

It does look like this test would become redundant, but so does the whole `--strict` + `SKIP_ERROR` logic of `sedfile`. You'd want to [revert that feature](https://github.com/docker-mailserver/docker-mailserver/commit/29c2d975ec40a8a7d26814b473fa0c44d2b95b6a#diff-39578133bff98a4e812dd8ac4469fa3e0ee924e7d4028f549a100170dad0647c) entirely then?

Reply:
You're probably, right; in one of the last PRs in the series after #3115, I will try to remeber this and maybe `sedfile` does not need an adjust at all; as you've said.

Reply:
> maybe `sedfile` does not need an adjust at all; as you've said.

I don't think it does :)

It would only needs changes if we wanted to remove any support we've added to (partially) handle starting the container with persisted internal state.

No harm in leaving that around for a while though, easier to have a single PR specifically removing just that if it were a maintenance concern, so that it's self-documenting of what would need to be restored if we did want to bring back support for restarting or restoring stopped containers.